pedpy/data/geometry.py

```diff
@@ -313,6 +314,16 @@ def line(self):
         """
         return self._line
 
+    def normal_vector(self):
+        """Returns a normalized normal vector of the line."""
```

Comment:
Docstring not complete, should als contain a description of the return value, e.g., 
```python
        """Measurement line as :class:`shapely.LineString`.
        Returns:
            Measurement line as :class:`shapely.LineString`.
        """
```